### DISCUSSION TITLE
Emails not sending on deployed container

### DISCUSSION THREAD

**Discussion by BlackIsBlack** [fief-dev/fief#348] [Q&A]
I am able to reach and send emails from a local instance of fief, however when using a deployed container on Google Cloud, It fails to send the email. No error is reported in the logs either, just the `uvicorn.protocols.http.httptools_impl:send:489 - x.x.x.x:0 - "POST /forgot HTTP/1.1" 200 - {}` 

I'm using POSTMARK, and have all egress ports open. Is there perhaps something I'm missing in terms of firewall setup?

**Comment by fief-bailiff** [comment]
Hail, @BlackIsBlack 👋 Welcome to Fief's kingdom!

Our team will get back to you very soon to help.

In the meantime, take a minute to [star our repository](https://github.com/fief-dev/fief) ⭐️

<p align="center">
<img width="656" alt="star-fief" src="https://github.com/fief-dev/fief/assets/1144727/b2db7f59-102f-47e5-9751-67fa10372c52">
</p>

## Want to support us?

Subscribe to one of our paid plan to help us continue our work and receive exclusive information and benefits! Starts at $5/month 🪙

<p align="center">
    <a href="https://polar.sh/fief-dev">
    <picture>
      <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/embed/subscribe.svg?org=fief-dev&darkmode=1">
      <img alt="Subscribe" src="https://polar.sh/embed/subscribe.svg?org=fief-dev">
    </picture>
    </a>
</p>

Farewell!

**Comment by frankie567** [comment]
Hi @BlackIsBlack 👋 

So, you confirm it works with the same Postmark setup in local, but not in Google Cloud; that's right? If yes, that's indeed strange. Postmark provider uses a standard HTTP API, so if egress ports 80/443 are opened, then it should work.

The emails are sent by the worker process, so I suggest to have a look at the worker logs. We'll maybe have more clues about what's going on.
